sdks/java/extensions/google-cloud-platform-core/src/main/java/org/apache/beam/sdk/util/RetryHttpRequestInitializer.java

```diff
@@ -73,7 +73,9 @@ public boolean handleIOException(HttpRequest request, boolean supportsRetry)
       if (willRetry) {
         LOG.debug("Request failed with IOException, will retry: {}", request.getUrl());
       } else {
-        LOG.warn("Request failed with IOException, will NOT retry: {}", request.getUrl());
+        LOG.warn(
```

Comment:
Rephrased the message as suggested.

Counting number of retries seems indeed not practical in this PR.